README.md

```diff
@@ -103,66 +105,66 @@ qjs >
 
 [👋 Missing a language?](https://github.com/wasmerio/wasmer/issues/new?assignees=&labels=%F0%9F%8E%89+enhancement&template=---feature-request.md&title=)
 
-[rust logo]: ./assets/languages/rust.svg
+[rust logo]: https://raw.githubusercontent.com/wasmerio/wasmer/master/assets/languages/rust.svg
```

Comment:
Using absolute urls are better since that way the READMEs will render properly anywhere